ipn/ipnlocal/local.go

```diff
@@ -1303,6 +1303,11 @@ func (b *LocalBackend) updateFilterLocked(netMap *netmap.NetworkMap, prefs ipn.P
 			localNetsB.AddPrefix(p)
 		}
 		packetFilter = netMap.PacketFilter
+
+		if packetFilterPermitsUnlockedNodes(netMap.Peers, packetFilter) {
+			b.logf("WARNING: server sent invalid packet filter permitting traffic to unlocked nodes; rejecting all packets now")
```

Comment:
Should probably be a health report too, since hitting this condition breaks the data plane.

Reply:
done